test/unit.html

```diff
@@ -28,11 +28,9 @@
 
       mocha.setup({
         ui: 'tdd',
-        reporter: post_xunit_to ? 'xunit' : 'html'
+        reporter: 'html',
+        bail: false
       });
-
-      var xunit = '';
-      Mocha.process.stdout.write = function(line) { xunit += line; };
```

Comment:
Desmos made some probably incompatible changes here to integrate our branch with Jenkins instead of circle CI.

I don't think we want to merge this upstream. I think we either need to get `desmosinc/mathquill` on CircleCI instead of Jenkins, or else we need to make this configurable enough that you can choose which project you want to use.

Reply:
Note: initial CI changes to support Jenkins landed here: https://github.com/desmosinc/mathquill/pull/134

Reply:
Resolved in https://github.com/desmosinc/mathquill/pull/166